src/common/form/types.js

```diff
@@ -0,0 +1,22 @@
+// @flow
+
+export type Field = {|
+  error: ?string,
+  touched: boolean,
+  value: any,
```

Comment:
Maybe use `mixed` or `string | number` here if possible.
It might not be possible but this could be `Field<TType>` as well.

Reply:
I'll figure about mixed or TType, but `value` is definitely not just string or number, it can be literally whatever. string for a  textfield, boolean for a radiogroup or checkbox, a date for a datapicker, object for some custom-weird-form-component

Reply:
Ok, sounds good.  Just move forward with `any` for now and maybe we can look into this later if it makes sense.